src/main/webapp/style.css

```diff
@@ -192,4 +192,16 @@ search-button {
   margin-top: 20px;
   padding: 10px 20px;
   text-align: center;  
+}
+
+li.pointer {
+  cursor: pointer;
+}
+
+li:hover {
+  background-color: rgba(255, 235, 153, 0.328);
```

Comment:
We want all of the colors to be uniform, so you should change this to the hex number. Make sure to make the letters lowercase.